import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java

```diff
@@ -92,6 +92,32 @@
   )
   String monitoringProjectId = "";
 
+  @Parameter(
+      names = {"--tag_edit_remove"},
+      description = "Tags to remove during C-STORE upload, comma separated. Only one of 'tag_edit' group flags may be present"
+  )
+  String tagsToRemove = "";
+
+  @Parameter(
+      names = {"--tag_edit_keep"},
+      description = "Tags to keep during C-STORE upload, comma separated. Only one of 'tag_edit' group flags may be present"
+  )
+  String tagsToKeep = "";
+
+  @Parameter(
+      names = {"--tag_edit_profile"},
+      description = "Filter tags by predefined profile during C-STORE upload. Only one of 'tag_edit' group flags may be present. "
+      + "Values: TAG_FILTER_PROFILE_UNSPECIFIED, CHC_BASIC"
```

Comment:
Is TAG_FILTER_PROFILE_UNSPECIFIED an actual possible value a user would specify or just a default?

Reply:
This part is quite literal, wrong value will fail adapter startup like this:
java.lang.IllegalArgumentException: No enum constant com.google.cloud.healthcare.deid.redactor.protos.DicomConfigProtos.DicomConfig.TagFilterProfile.NO_SUCH_VALUE

Reply:
Sure, but I don't think users will ever want to specify TAG_FILTER_PROFILE_UNSPECIFIED here. Right?

Reply:
Ok, I'll remove the mention here (it does same thing as CHC_BASIC anyway).